Cannot compile Fabric 5.92.0 (or later) when using AMD as TypeScript target

<!-- delete this template for feature requests -->

### Bug Report

- __Package version(s)__: 5.92.0 (or later)
- __Browser and OS versions__: 

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Blocking

Products/sites affected: (if applicable) Microsoft Flow, Logic Apps

#### Describe the issue:

This change in `/packages/styling/src/styles/getGlobalClassNames.ts` (line 1) breaks AMD builds due to the unresolvable import.

````ts
import { ITheme } from '../interfaces';
````

should be

````ts
import { ITheme } from '../interfaces/index';
`````

#### Actual behavior:

Build fails with error:
> node_modules/@&#xfeff;uifabric/styling/lib-amd/styles/getGlobalClassNames.d.ts(1,24): error TS2307: Cannot find module '../interfaces'.

#### Expected behavior:

Build completes successfully.

--- a/common/changes/@uifabric/styling/fix-amd-issue_2018-05-16-19-48.json
+++ b/common/changes/@uifabric/styling/fix-amd-issue_2018-05-16-19-48.json
@@ -1,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "Fix import to be compatible with AMD bundlers",
+      "type": "patch"
+    }
+  ],
+  "packageName": "@uifabric/styling",
+  "email": "cliff.koh@microsoft.com"
+}

--- a/packages/styling/src/styles/getGlobalClassNames.ts
+++ b/packages/styling/src/styles/getGlobalClassNames.ts
@@ -1,4 +1,4 @@
-import { ITheme } from '../interfaces';
+import { ITheme } from '../interfaces/index';
 
 export type GlobalClassNames<IStyles> = Record<keyof IStyles, string>;
 

